tests/test_bs.py

```diff
@@ -0,0 +1,229 @@
+import pytest
+import yaml
+import time
+import re
+import subprocess
+from os import path
+from kubernetes import client, config
+from node_info import NodeInfo
+from pytest_testconfig import config as testconfig
+
+from elasticsearch import Elasticsearch
+from elasticsearch_dsl import Search, Q
+
+BOOT_DEPLOYMENT_FILE='./k8s/bootstrap.yml'
+CLIENT_DEPLOYMENT_FILE = './k8s/client.yml'
+
+
+def update_bootsrap_node_in_client(yml_input, bs_ip, bs_port, bs_key):
+    yml_input['spec']['template']['spec']['containers'][0]['args'][-1] = "{0}:{1}/{2}".format(bs_ip, bs_port, bs_key)
+    return yml_input
+
+
+def wait_to_deployment_to_be_ready(deployment_name, name_space):
+    total_sleep_time = 0
+
+    while True:
+        resp = client.AppsV1Api().read_namespaced_deployment(name=deployment_name, namespace=name_space)
+        if not resp.status.unavailable_replicas:
+            print("Total time waiting for deployment {0}: {1} sec".format(deployment_name, total_sleep_time))
+            break
+        time.sleep(1)
+        total_sleep_time += 1
+
+
+def create_deployment(file_name,name_space, **kwargs):
+    with open(path.join(path.dirname(__file__), file_name)) as f:
+        dep = yaml.safe_load(f)
+        if kwargs:
+            dep = update_bootsrap_node_in_client(dep,
+                                                 kwargs['bootstrap_ip'],
+                                                 kwargs['bootstrap_port'],
+                                                 kwargs['bootstrap_key'])
+
+        k8s_beta = client.ExtensionsV1beta1Api()
+        resp1 = k8s_beta.create_namespaced_deployment(body=dep, namespace=name_space)
+        wait_to_deployment_to_be_ready(resp1.metadata._name, name_space)
+
+        return resp1
+
+
+def delete_deployment(deployment_name, name_space):
+    k8s_beta = client.ExtensionsV1beta1Api()
+    resp = k8s_beta.delete_namespaced_deployment(name=deployment_name,
+                                                 namespace=name_space,
+                                                 body=client.V1DeleteOptions(
+                                                     propagation_policy='Foreground',
+                                                     grace_period_seconds=5))
+    return resp
+
+
+def query_bootstrap_es(indx, namespace, bootstrap_po_name):
```

Comment:
this name seems too similar to the method below but it's really different. consider better naming